src/components/navigation/navbar/common/profile/ProfileMenuBody.vue

```diff
@@ -36,11 +31,7 @@
         },
         setup() {
             function logOut() {
-                const store = useStore();
-                store.commit(MutationTypes.SET_LOGINDATA, { username: "", password: "" });
-                store.commit(MutationTypes.SET_USER, {} as Student | Lecturer | Admin);
-                store.commit(MutationTypes.SET_ROLE, Role.NONE);
-                store.commit(MutationTypes.SET_LOGGEDIN, false);
+                logout();
```

Comment:
Maybe we should directly call the logout method in the template instead of having a `logOut` and `logout` method. Otherwise, rename one of them.